apps/web-giddh/src/app/invoice/preview/invoice.preview.component.ts

```diff
@@ -534,53 +538,46 @@ export class InvoicePreviewComponent implements OnInit, OnChanges, OnDestroy {
 
                         } else {
                             this.datePickerOptions = {
-                                ...this.datePickerOptions, startDate: moment(a[0], GIDDH_DATE_FORMAT).toDate(),
-                                endDate: moment(a[1], GIDDH_DATE_FORMAT).toDate(),
-                                chosenLabel: a[2]
+                                ...this.datePickerOptions, startDate: moment(dateObj[0], GIDDH_DATE_FORMAT).toDate(),
```

Comment:
Not sure of the use of this since we are using new datepicker now. Please confirm